### ISSUE TITLE
Very high load / block process bookticker with last version - blocked process

### ISSUE THREAD

**Issue by TKR-US** [sammchardy/python-binance#870]
Hello,

i've a suprise with the last version of library, about bookticker "!bookTicker"
the problem is veru strange, but it is very easy to reproduct it

```
twm = ThreadedWebsocketManager(api_key=api_key, api_secret=api_secret)
twm.start()`

def process_message_bookticker(msg):
    try:
        if msg['data']['s'].endswith("ETHBTC") == True:
            pprint(msg['data'])

    except NameError as e:
        print("bookticker EMPTY")
        print(e)
        twm.stop()


twm.start_multiplex_socket(callback=process_message_bookticker, streams=['!bookTicker'])
```

So, just wait.. 30/40 secondes, maximum 1 minute, and you see a freeze of print data.. and processor to 100% load..
After this freeze, print start again during... 30/40 secondes, and freeze again... ( so i loose in real time some informations :( )

idea?

Python3 under ubuntu 20.

**Comment by TKR-US** [comment]
ok, i observe  with "!miniTicker@arr", not a complete freeze, but big slowdowns..

it's a real problem :(

**Comment by TKR-US** [comment]
i roolback to 0.7.11 pending the correction.

I am willing to do additional tests if necessary

Regards

**Comment by kokojiji01** [comment]
Did you get it working with the older releases? If the problem is only occurring with the most recent release, I could also try to rollback.

**Comment by TKR-US** [comment]
Yep, on 0.7.11 it's works fine..
but.. but.. i observe slowdowns, but not freeze..

i don't know if the problem is :
- this library ( i'm sure no..)
- the websocket library
- Binance API

as soon as I have a little time, I would do this simple test with the go or php library to see if I observed the same behavior

**Comment by kokojiji01** [comment]
Let me know what you find out. I'm having the same issue and would also like to find out where the problem is.

**Comment by TKR-US** [comment]
After investigations, i confirm there is no problem and no latency with 0.7.11..

Waiting for correction about the new version :)

**Comment by sammchardy** [comment]
Thanks for the example @TKR-US will investigate

**Comment by Nise-2-meet-U** [comment]
same here

**Comment by sammchardy** [comment]
This has now been fixed in v1.0.11

**Comment by TKR-US** [comment]
Hello,

Thanks for update, it works !

Regards
